.github/workflows/build-bootc-image.yaml

```
@@ -0,0 +1,152 @@
+          echo "OCI_IMAGE=${{ inputs.oci-registry }}/${{ inputs.oci-org }}/${{ inputs.demo }}" >> $GITHUB_ENV
+          echo "OCI_IMAGE_TAG=${GITHUB_SHA::8}" >> $GITHUB_ENV
+
+      - name: Create manifest list
+        run: |
+          podman manifest create ${OCI_IMAGE}:${OCI_IMAGE_TAG}
+          for arch in amd64 arm64; do
+            podman manifest add ${OCI_IMAGE}:${OCI_IMAGE_TAG} docker://${OCI_IMAGE}:${OCI_IMAGE_TAG}-${arch}
+          done
+
+      - name: Tag manifest list
+        if: {{ inputs.tags != '' }}
```

actionlint (1.7.4)

expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)